calour/heatmap/heatmap.py

```diff
@@ -490,13 +497,13 @@ def plot(exp: Experiment, title=None,
         for each column. It doesn't plot color bars by default (None)
     barx_width : float or list of float, optional
     bary_width : float or list of float, optional
-        The thickness of the each bar. The default thickness usually looks good enough.
+        The thickness of the each bar along x axis or y axis. The default thickness usually looks good enough.
     barx_colors : dict, matplotlib.colors.ListedColormap, optional
     bary_colors : dict, matplotlib.colors.ListedColormap, optional
         The colors for each unique values in the column of sample/feature metadata
-    barx_label : bool, optional
-    bary_label : bool, optional
-        whether to show the labels on the bars.
+    barx_label : bool or list of bool, optional
+    bary_label : bool or list of bool, optional
+        whether to show the labels on the bars along x axis or y axis.
```

Comment:
What does list of bool do? Is it one per bar?

Reply:
correct